regression/goto-synthesizer/loop_contracts_synthesis_05/main.c

```diff
@@ -0,0 +1,48 @@
+#include <assert.h>
+#include <stdlib.h>
+
+#define SIZE 8
+
+struct blob
+{
+  char *data;
+};
+
+void main()
+{
+  struct blob *b = malloc(sizeof(struct blob));
```

Comment:
Maybe insert some guards for the case where malloc returns NULL? This isn't default CBMC behavior now, but may become so in a subsequent major release (and will be disabled by an option rather than enabled).